internal/resources/policy/scope/scope_schema.go

```diff
@@ -22,7 +22,7 @@ import (
 var (
 	ScopeSchema = &schema.Schema{
 		Type:        schema.TypeList,
-		Description: "Scope for the custom, security, image and namespace quota policy, having one of the valid scopes for custom, security and namespace quota policy: cluster, cluster_group or organization and valid scopes for image policy: workspace or organization.",
+		Description: "Scope for the custom, security, image, network and namespace quota policy, having one of the valid scopes for custom, security and namespace quota policy: cluster, cluster_group or organization and valid scopes for image and network policy: workspace or organization.",
```

Comment:
nit: having one of the valid scopes for custom, security`, image, network` and namespace quota policy:

Reply:
I don't see the difference in what you have suggested and what is present in this change. I even tried to copy paste your suggestion, still there is no difference.

Reply:
I have wrongly read the description but honestly the description can be much better than this